pkg/datapath/linux/config/config.go

```diff
@@ -103,6 +104,12 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 			ipv4NP := node.GetNodePortIPv4()
 			cDefinesMap["IPV4_NODEPORT"] = fmt.Sprintf("%#x", byteorder.HostSliceToNetwork(ipv4NP, reflect.Uint32).(uint32))
 		}
+
+		if option.Config.EnableIPv4FragmentsTracking {
+			cDefinesMap["IPV4_FRAGMENTS"] = "1"
+			cDefinesMap["IPV4_FRAG_DATAGRAMS_MAP"] = "cilium_ipv4_frag_datagrams"
```

Comment:
If you export `fragmap.mapName` you could then use it here.

Reply:
Oh, right, thanks for pointing this out.